R/mockForestPlotData.R

```diff
@@ -0,0 +1,31 @@
+#' Function to create mock forest plot data.
+#'
+#' @param seed Seed value for the random mock data generated
+#' @param numRows Number of rows
+#'
+#' @export
+#'
+#' @examples
+#' \donttest{
+#' library(ggoxford)
+#'
+#' mockForestPlotData()
+#' }
+#'
+mockForestPlotData <- function(seed = 1, numRows = 10) {
+  set.seed(seed = seed)
+  res <- dplyr::tibble(
+    group_name = paste0("Outcome ", 1:numRows),
+    group_level = "Cohort 1",
+    strata_name = "Overall",
+    strata_level = "Overall",
+    variable = paste0("Variable ", 1:numRows),
+    variable_level = NA,
+    variable_type = "numeric",
+    estimate = round((stats::runif(n = numRows, min = 0.51, max = 3.01)),2),
+    estimate_type = "95% confidence interval"
+  )
+  res <- res |> dplyr::mutate(estimate = paste0("(",estimate,", ",(estimate - round((stats::runif(n = numRows, min = 0.25, max = 0.75)),2)),", ",(estimate + round((stats::runif(n = numRows, min = 0.25, max = 0.75)),2)),")"))
```

Comment:
so the idea is that each estimate type would have a different row, so for each variable we would have 3 rows.